.env.example

```
@@ -0,0 +1,6 @@
+DB_HOST =
+DB_PORT =
+DB_USERNAME =
+DB_PASSWORD =
+DB_NAME =
+NODE_ENV =
```

```suggestion
DB_HOST =localhost
DB_PORT =3306
DB_USERNAME = TODO
DB_PASSWORD = TODO
DB_NAME = TODO
```
NODE_ENV là do webpack truyền vào rồi, ko cần insert vào đây